docs/debugging-openshift.md

```diff
@@ -99,6 +99,21 @@ If it's not running, you can launch it via:
 
     $ oadm registry --create --credentials="${KUBECONFIG}"
 
+Insecure Docker Registry
+------------------------
+
+If you have problems when pushing to integrated registry, for example following failures in build logs:
+
+    E1124 14:51:23.828346       1 dockerutil.go:74] push for image 172.30.216.184:5000/test/image:latest failed, will retry in 5s seconds ...
+    ...
+    F1124 14:51:28.828654       1 builder.go:60] Build error: Failed to push image. Response from registry is: unable to ping registry endpoint https://172.30.216.184:5000/v0/
+    v2 ping attempt failed with error: Get https://172.30.216.184:5000/v2/: tls: oversized record received with length 20527
+     v1 ping attempt failed with error: Get https://172.30.216.184:5000/v1/_ping: tls: oversized record received with length 20527
+
+If you are running Docker as a service via `systemd`, add the `--insecure-registry 172.30.0.0/16` argument to the options value in `/etc/sysconfig/docker` and restart the Docker daemon.  Otherwise, add "--insecure-registry 172.30.0.0/16" to the Docker daemon invocation, eg:
```

Comment:
Perhaps a note about where to find this value would be helpful (`networkConfig.serviceNetworkCIDR` option in `master-config.yaml`).